test/integration/dashboard_test.rb

```diff
@@ -46,4 +46,12 @@ def assert_dashboard_link(text)
   test "dashboard link hosts with alerts disabled" do
     assert_dashboard_link 'Hosts with alerts disabled'
   end
+
+  test 'widgets not in dashboard show up in list' do
+    users(:admin).widgets = users(:admin).widgets[1..-1]
```

Comment:
Can you change this to specifically remove "Status table" from the user's widgets, or to remove the last and then check for its name in the list?  It looks like it'd break if another was ever added.